dac/daemon/src/main/java/com/dremio/dac/cmd/upgrade/UpgradeTask.java

```diff
@@ -55,14 +66,35 @@ public String getDescription() {
     return dependencies;
   }
 
+  /**
+   * Gets the upgrade task name.
+   *
+   * @return the upgrade task name
+   */
   String getTaskName() {
     return taskName;
   }
 
+  /**
+   * Gets the upgrade task UUID.
+   *
+   * @return the UUID from the current task
+   */
   public abstract String getTaskUUID();
 
+  /**
+   * Executes an upgrade task.
```

Comment:
I think we should explain what is an upgrade task

Reply:
resolve